### PR TITLE
Add Svelte wrappers

### PR DESCRIPTION
**Pull request by metonym**
This feature implements `@carbon/charts` in Svelte to serve as a companion library for [carbon-components-svelte](https://github.com/IBM/carbon-components-svelte).

### Updates
- add `svelte` package under `packages/`
- add wrappers for Svelte
- use fixtures from `@carbon/charts/demo/data` to generate Storybook stories
- adjust build script for library to copy `src` to `dist` for Svelte component entry

### Demo screenshot or recording

Screenshot of the Storybook.

Screen Shot 2020-04-10 at 9 55 40 AM

### Review checklist (for reviewers only)
- [ ] Demos all features
- [ ] Documented/annotated
- [ ] Matches UI/UX specs
- [ ] Meets the [code style guide](https://github.com/IBM/carbon-charts/wiki/Code-style-guide)
- [ ] Accessible
- [ ] Mobile first (responsive)
- [ ] RTL support (bidirectional text)
- [ ] Performant (limited bloat)

### CODE REVIEW

File: packages/svelte/.babelrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"presets": [["env", { "modules": false }]],
```
Comment (theiliad) [inline packages/svelte/.babelrc:2]: we should break this in multiple lines

File: packages/svelte/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+yarn add @carbon/charts @carbon/charts-svelte d3
+```
+
+## Step-by-step instructions
+
+TBA
+
+## Codesandbox examples
+
+TBA
```
Comment (theiliad) [inline packages/svelte/README.md:30]: I've updated the READMEs for packages. We should do the same here

File: packages/svelte/rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+		input: "./src/index.js",
+		output: {
+			file: "./dist/bundle.js",
+			format: "umd",
+			name: "ChartsSvelte",
+			globals: {
+				"@carbon/charts": "Charts"
+			}
+		},
+		plugins: [svelte(), resolve(), terser()],
```
Comment (theiliad) [inline packages/svelte/rollup.config.js:16]: we should break this in multiple lines

File: packages/svelte/package.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+		"img-loader": "2.0.0",
+		"rollup": "1.27.10",
+		"rollup-plugin-commonjs": "10.1.0",
+		"rollup-plugin-node-resolve": "5.2.0",
+		"rollup-plugin-svelte": "^5.2.1",
+		"rollup-plugin-terser": "5.1.2",
+		"sass": "1.25.0",
+		"sass-loader": "8.0.0",
+		"style-loader": "0.19.0",
+		"svelte": "^3.20.x",
```
Comment (theiliad) [inline packages/svelte/package.json:75]:
```suggestion
		"svelte": "3.20.x",
```

File: packages/svelte/package.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+		"svelte-loader": "^2.13.6",
```
Comment (theiliad) [inline packages/svelte/package.json:76]:
```suggestion
		"svelte-loader": "2.13.6",
```

File: packages/svelte/package.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+	"dependencies": {
+		"@carbon/charts": "^0.30.10"
+	},
+	"peerDependencies": {
+		"svelte": "^3.20.x"
+	},
+	"devDependencies": {
+		"@storybook/addon-knobs": "5.3.12",
+		"@storybook/addon-options": "5.3.12",
+		"@storybook/svelte": "^5.3.18",
```
Comment (theiliad) [inline packages/svelte/package.json:57]:
```suggestion
		"@storybook/svelte": "5.3.18",
```

**Comment by ninja511** [comment]
"svelte": "^3.20.x" is the only open source of relevance, and it's MIT, which is favorable to IBM. I just need to add new AWSOM approval for it. In the mean time, you can go ahead and proceed. Thanks.

File: packages/svelte/README.md (hunk 2 above)
Comment (metonym) [reply packages/svelte/README.md]: I've filled in the Codesandbox examples sections in the README.

However, the "Step-by-step" section is still `TBA`. Should I add in a few steps for basic usage?

Comment (theiliad) [reply packages/svelte/README.md]: hmmm... yeah maybe at least in the README just add that

**Comment by theiliad** [comment]
Couple of points @metonym:
- would you be able to add codesandbox demos to the svelte stories? (you should be able to reference the vanilla/react stories on how they're being done, and here's a svelte codesandbox sample app https://codesandbox.io/s/svelte)
- I added a netlify app to build svelte deploy previews, should be good to go as long as you've setup similar NPM scripts in your `package.json` as the other packages
- we need to add a piece about the ownership of this work in terms of maintenance. I'm thinking of a `maintainers` field within `package.json` and a piece in the README pointing people towards you. I'll write up something for the README soon

**Comment by metonym** [comment]
@theiliad 

1. Added CodeSandbox demos for Svelte
2. The npm scripts are identical to those of the react package
3. Added maintainers, contributors to `svelte/package.json`

File: packages/svelte/package.json [hunk 7]
```diff
@@ -0,0 +1,97 @@
+{
+	"name": "@carbon/charts-svelte",
+	"version": "0.30.10",
```
Comment (theiliad) [inline packages/svelte/package.json:3]:
```suggestion
	"version": "0.30.18",
```

**Pull request merged**